ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-commands.sh

```diff
@@ -20,10 +20,10 @@ metadata:
   name: ${CLUSTER_NAME}
 platform:
   openstack:
+    apiFloatingIP:    ${LB_FIP_IP}
     cloud:            ${OS_CLOUD}
-    externalNetwork:  ${OPENSTACK_EXTERNAL_NETWORK}
     computeFlavor:    ${OPENSTACK_COMPUTE_FLAVOR}
-    lbFloatingIP:     ${LB_FIP_IP}
+    externalNetwork:  ${OPENSTACK_EXTERNAL_NETWORK}
```

Comment:
cosmetic change. Only need to change lbFloatingIP -> apiFloatingIP. This many changes make the commit hard to read when exploring the branch? for example "git show this commit" willl create a very large output, when the only thing that change was lbFloatingIP->apiFloatingIP